specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/subnetResources.json

```diff
@@ -233,6 +233,10 @@
         "status": {
           "$ref": "#/definitions/SubnetState",
           "description": "Connection State of the subnet."
+        },
+        "systemData": {
```

Comment:
Same here, the wrong place to have systemData.

Reply:
Thanks! Updated